src/components/numberpad/NumberPad.tsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable functional/immutable-data */
+import React, { ComponentProps, useRef } from "react";
+import { View } from "react-native";
+import { BiometricsValidType } from "../../utils/types";
+import { IONumberPadButtonStyles, IOStyles } from "../../core";
+import { VSpacer } from "../spacer";
+import { IconButton } from "../buttons";
+import { IOIconSizeScale, IOIcons } from "../icons";
+import { NumberButton } from "./NumberButton";
+
+type BiometricAuthProps =
```

Comment:
can this type be just :
```
type BiometricAuthProps = {
  biometricType?: BiometricsValidType;
  onBiometricPress?: () => void;
  biometricAccessibilityLabel?: string;
};
```

Reply:
Using the union type here defined we can handle the constraint of having all three props none of them defined at the same time: if we have a biometric option the tercet must be valorized correctly